README.md

```diff
@@ -43,7 +87,6 @@ Run `ng g @nrwl/angular:lib my-lib` to generate a library.
 
 Libraries are shareable across libraries and applications. They can be imported from `@backbase/mylib`.
```

Comment:
```suggestion
Libraries can be shared across libraries and applications. You can import them from `@backbase/mylib`.
```
It is not clear what you want to say with the second sentence. Do you mean: "you can import other libraries from  `@backbase/mylib` for other apps?
Or do you mean that you can access other libraries, and "share" them (i.e. means not having a local version of the library for the app?"